src/platformdirs/unix.py

```diff
@@ -139,6 +155,27 @@ def _first_item_as_path_if_multipath(self, directory: str) -> Path:
         return Path(directory)
 
 
+def get_user_dirs_folder(key: str) -> Optional[str]:
+    """Return directory from user-dirs.dirs config file. See https://freedesktop.org/wiki/Software/xdg-user-dirs/"""
+    user_dirs_config_path = os.path.join(Unix().user_config_dir, "user-dirs.dirs")
+    if not os.path.exists(user_dirs_config_path):
+        return None
```

Comment:
in this case I think will be easier to read if you don't return early just nest the checks